### ISSUE TITLE
McAffe bloatware

### ISSUE THREAD

**Issue by betelopson** [felixrieseberg/windows95#147]
I don`t get it, how this application is available here when a bloatware McAfee (`antivirus`) automatically in the background installs itself. Not cool.

**Comment by workdowg** [comment]
I don't understand your issue... Nothing like that got installed anytime I've done this.

**Comment by betelopson** [comment]
Some sort of mcafee installed it self on that pc after I'd started the .exe version of this 95. I realized this after I'd started the .exe version and than the pc was restarted on the next day. I wouldn't have figured this out if it'd not started it self automatically. It was an easy uninstall, and the firewall I've been using on that machine blocked the communication for this application and since than wasn't any problem and looks like it's completely gone. I know it looks very strange but I really pay close attention on that pc and I try to avoid any problem on it. But this is just one man's opinion I'd be wrong but I'm sure of it I'm not willing to start it again I deleted that version since than. I'm sure the other options doesn't have any problem like this. I just wanted to pointed out and I din't expected this either.
